build-support/bin/check_header_helper.py

```diff
@@ -16,6 +16,7 @@
 import sys
 from io import open
 
+
```

Comment:
Extra space?

Reply:
Yes. Required to get `./pants fmt build-support::` working.